src/GitHub.App/Services/ImageDownloader.cs

```diff
@@ -30,7 +30,6 @@ public IObservable<byte[]> DownloadImageBytes(Uri imageUri)
                 BaseAddress = new Uri(imageUri.GetLeftPart(UriPartial.Authority)),
                 Endpoint = new Uri(imageUri.PathAndQuery, UriKind.RelativeOrAbsolute),
                 Method = HttpMethod.Get,
-                AllowAutoRedirect = true
```

Comment:
This is a side-effect of the repository redirects from the API, and because `AllowAutoRedirect` doesn't set the `Authorization` header [citation](https://msdn.microsoft.com/en-us/library/system.net.httpwebrequest.allowautoredirect%28v=vs.110%29.aspx) when redirecting, it would break for most of our scenarios.

> The Authorization header is cleared on auto-redirects and HttpWebRequest automatically tries to re-authenticate to the redirected location. In practice, this means that an application can't put custom authentication information into the Authorization header if it is possible to encounter redirection.

We're now handling the redirects in Octokit and setting the right authorization headers. See https://github.com/octokit/octokit.net/pull/808 for the whole backstory.